### DISCUSSION TITLE
please add the windows 10 search menu

### DISCUSSION THREAD

**Discussion by V7614** [valinet/ExplorerPatcher#3557] [Ideas]
I wanted my windows 11 to look like 10 but windows search is still windows 11 can you add the option

**Comment by pyrates999** [comment]
According to this, it's not possible in windows 11: https://github.com/valinet/ExplorerPatcher/issues/3539#issuecomment-2271589197

**Comment by milestprower92** [comment]
> I wanted my windows 11 to look like 10 but windows search is still windows 11 can you add the option

not possible on 22H2+

**Reply by V7614** [reply]
thanks for the information
